test/bdd/features/get-errors.feature

```diff
@@ -6,19 +6,19 @@ Feature: Get errors test
   @get-errors
   Scenario: Getting non existent UAL
     Given I setup 4 nodes
-    And I wait for 10 seconds
+    And I wait for 2 seconds
     And I call get directly to ot-node 1 with nonExistentUAL
     And I wait for last resolve to finalize
-    Then Last GET operation finished with status: GetLocalError
+    Then Last GET operation finished with status: GetAssertionIdError
```

Comment:
@djordjekovac changed this `GetAssertionIdError` to `GetLocalError` in his PR that fixed BDD tests. So I want just to confirm that you have latest changes from develop and that this is intentional revert?

Reply:
I changed it back because when receiving the a non existent UAL, the get latest assertion id command should fail, and this error should be thrown. The get local command should not be executed in that case.